### PR TITLE
Added Supplier statement model

### PR DESCRIPTION
**Pull request by make-it-git**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #55 (partially)

Notify the following people: @darrynten

### CODE REVIEW

**Comment by darrynten** [comment]
This looks dependant on #59 can you confirm if that is the case @make-it-git?

**Comment by make-it-git** [comment]
Yes, I used some things implemented in that PR

File: src/Models/SupplierStatement.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    ];
+
+    /**
+     * @var array $featureMethods
+     */
+    protected $featureMethods = [
+        'all' => 'POST',
+        'get' => 'GET',
+        'save' => 'POST',
+        'delete' => 'DELETE'
```
Comment (darrynten) [inline src/Models/SupplierStatement.php]: There are some missing trailing commas in this file 😊

File: src/Models/SupplierStatementLine.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            'nullable' => true,
+            'readonly' => true,
+        ],
+        'credit' => [
+            'type' => 'double',
+            'nullable' => true,
+            'readonly' => true,
+        ],
+    ];
+}
```
Comment (darrynten) [inline src/Models/SupplierStatementLine.php:74]: I was wondering when we would have one with no calls :D

**Review by darrynten — CHANGES_REQUESTED** [review]
Only some missing trailing commas in SupplierStatement, but the rest looks great!

**Review by darrynten — APPROVED** [review]

**Review by darrynten — CHANGES_REQUESTED** [review]
@make-it-git it just needs the README to be updated with the models checked off the list

**Review by darrynten — APPROVED** [review]

**Pull request merged**